src/Tools/dotnet-trace/Extensions.cs

```diff
@@ -22,6 +56,29 @@ public static List<EventPipeProvider> ToProviders(string providers)
                 new List<EventPipeProvider>() : providers.Split(',').Select(ToProvider).ToList();
         }
 
+        public static EventPipeProvider ToCLREventPipeProvider(string clreventslist)
+        {
+            if (clreventslist == null || clreventslist.Length == 0)
+            {
+                return null;
+            }
+
+            var clrevents = clreventslist.Split("+");
+            long clrEventsKeywordsMask = 0;
+            for (var i = 0; i < clrevents.Length; i++)
+            {
+                if (CLREventKeywords.TryGetValue(clrevents[i], out var keyword))
+                {
+                    clrEventsKeywordsMask |= keyword;
+                }
+                else
+                {
+                    throw new ArgumentException($"{clrevents[i]} is not a valid CLR event keyword");
+                }
+            }
+            return new EventPipeProvider(CLREventProviderName, (EventLevel)4, clrEventsKeywordsMask, null);
```

Comment:
Should the `EventLevel` be configurable?

Reply:
Oops, the "dead code" above is supposed to handle this - will update this. Thanks for catching that!